src/components/VSelect/VSelect.vue

```diff
@@ -329,7 +329,21 @@
           click: () => {
             if (!this.isActive) this.isActive = true
           },
-          keydown: this.onKeyDown // Located in mixins/autocomplete.js
+          keydown: (e) => {
+            if (!this.isActive &&
+                !this.multiple &&
```

Comment:
This is the only part that confuses me. Is there a reason this shouldn't be supported on multiple?